api/searchCases/index.ts

```diff
@@ -144,14 +178,20 @@ export function handleElasticSearchResponse(proxyRes, req, res, json): {} {
       return caseObj
     })
 
-    const handledResponse = {
+    return {
       'columns': json.headers[0].fields,
       'results': results,
       'total': json.total,
     }
-
-    return handledResponse
-  } else {
-    return {}
   }
+  return {}
+}
+
+function canApplyWildCardSearch(wildcardSearchFields: { [key: string]: string[] }, caseType: string, criterion: string): boolean {
+  return wildcardSearchFields.hasOwnProperty(caseType) && wildcardSearchFields[caseType].indexOf(criterion) >= 0
+}
+
+function phraseHasSpecialCharacters(phrase: string): boolean {
+  const specialCharacters: string[] = [' ', '-', '_']
```

Comment:
This should also be declared outside of this instance method.

Reply:
@paul-graham moved the special character array outside the scope of the function.